dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/extension/SpringExtensionFactory.java

```diff
@@ -51,6 +53,16 @@ public static void clearContexts() {
     @Override
     @SuppressWarnings("unchecked")
     public <T> T getExtension(Class<T> type, String name) {
+
+        //SPI never get from spring bean factory
+        if (type.isInterface() && type.isAnnotationPresent(SPI.class)) {
+            ExtensionLoader<T> loader = ExtensionLoader.getExtensionLoader(type);

```

Comment:
in this case, we should simply return `null` since it is SpiExtensionFactory's duty.

Reply:
you are right, it's not SpringExtensionFactory's duty to find SPI instance.

Reply:
you are right, it's not SpringExtensionFactory's duty to find SPI instance.